### PR TITLE
Meal app

### PR DESCRIPTION
**Pull request by Basir-Mohammadi**
## Proposed changes

The aim of this pull request is to Create a restaurant applicarion project.

**The following setups were implemented**
- Use JavaScript to make websites dynamic and build basic single-page apps.
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use webpack.
- Apply JavaScript best practices and language style guides in code.
- Use AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code synchronous.
- Perform a code review for a team member.
- [Video presentation](https://drive.google.com/file/d/1uRAmNc65VcY2z7mJbhmda30WQmQB4NEs/view?usp=sharing)
## Type of change

-   New feature

## Checklist

-    [Linting tests](https://github.com/microverseinc/linters-config) pass locally with my changes
-    Used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-    Documentation is done in a [professional

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
-# Capstone_Javascript
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Resturant Website Application
+
+> This is a website that allows users to look, like and comment on their favourite meal.
```
Comment (arslanbisharat) [inline README.md:5]: - [ ] nce you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

-     A description of the project.
-     A demo of the project features.
-     You should also highlight some interesting piece of code or something you built that you are very proud of.
-     You all should appear in the video and talk about the project.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @sajid-munawar  and @sajid-munawar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sajid-munawar) [reply README.md:5]: Hi @arslanbisharat, Thank you so much for the review and suggestions. We have added a video in the PR. Have you seen that?
Do we need to add that video to the README file?

**Review by Emmyn5600 — APPROVED** [review]
Hi @Basir-Mohammadi @sajid-munawar 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**